src/libraries/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -5773,10 +5772,10 @@ public void WriteNumberValueDecimal(bool formatted, bool skipValidation)
         [InlineData(true, false, "message", true)]
         [InlineData(false, true, "message", true)]
         [InlineData(false, false, "message", true)]
-        [InlineData(true, true, "mess><age", false)]
-        [InlineData(true, false, "mess><age", false)]
-        [InlineData(false, true, "mess><age", false)]
-        [InlineData(false, false, "mess><age", false)]
+        [InlineData(true, true, "mess><age", true)]
+        [InlineData(true, false, "mess><age", true)]
+        [InlineData(false, true, "mess><age", true)]
+        [InlineData(false, false, "mess><age", true)]
```

Comment:
With my changes, these were failing, as the test check for non-equality, but I think it's wrong. Let me know though.

Reply:
This was fine as it was before, it is asserting that the `expectedStr` is not equals to `keyString` when you use `UnsafeRelaxedJsonEscaping` which means that special characters (such as `>`, `<`) are not escaped. 

With your changes to `NormalizeToJsonNetFormat` you are always escaping `>` and `<`, so perhaps that's not the proper way of fixing https://github.com/dotnet/runtime/issues/32350.